architxt/database/export/relational.py

```diff
@@ -0,0 +1,328 @@
+import base64
+from datetime import datetime
+
+from sqlalchemy import (
+    BLOB,
+    Column,
+    Connection,
+    Date,
+    DateTime,
+    ForeignKey,
+    MetaData,
+    String,
+    Table,
+    insert,
+)
+from tqdm.auto import tqdm
+
+from architxt.schema import Schema
+from architxt.tree import NodeLabel, NodeType, Tree, has_type
+
+
+def export_relational(
+    conn: Connection,
+    *,
+    forest: list[Tree],
```

Comment:
`forest` is not an option and should be set as the main argument, connection may be a the second argument